Cross module tree shaking (#67)
rolldown-rs/rolldown: README.md

Fast JavaScript/TypeScript bundler in Rust with Rollup-compatible API.

# Rollup test status

> ![status](./packages/rollup-tests/test/status.svg)

Reviewer: ```suggestion
![status](./packages/rollup-tests/src/status.svg)
```

> ![status](https://github.com/rolldown/packages/rollup-tests/test/status.svg)

Reviewer: ```suggestion
![status](./packages/rollup-tests/test/status.svg)
```

> It is status for rollup function/file-hashes directory test cases.

Reviewer: I thought this is status of all tests of rollup. Why emphasize `function/file-hashes`?

Reviewer 2: Because other test case look like need diffrent format, we don't support it at now.

> It is status for rollup function and file-hashes directory test cases.

Reviewer: ```suggestion
It shows how many tests from rollup are passed by rolldown. The tests currently include the `function` and `file-hashes` directories of rollup tests.
```
